cmip6_downscaling/workflows/bcsd_flow.py

```diff
@@ -36,22 +37,40 @@
 
 # storage_prefix = config.get("runtime.cloud.storage_prefix")
 
+
 runtime = runtimes.get_runtime()
 
-intermediate_cache_store = CacheStore(
-    config.get("storage.intermediate.uri"),
-    storage_options=config.get("storage.intermediate.storage_options"),
-)
 
+# Transform Functions into Tasks -----------------------------------------------------------
 
-results_cache_store = CacheStore(
-    config.get("storage.results.uri"),
-    storage_options=config.get("storage.results.storage_options"),
-)
 
-# Transform Functions into Tasks -----------------------------------------------------------
+@task()
+def override_config(
+    intermediate_uri,
+    results_uri,
+    temporary_uri,
+    intermediate_storage_options,
+    results_storage_options,
+    temporary_storage_options,
+    runtime,
+    cleanup_flag,
+):
+    config.set(
+        {
+            'config.intermediate.uri': intermediate_uri,
+            'config.results.uri': results_uri,
+            'config.temporary.uri': temporary_uri,
+            'config.intermediate.storage_options': intermediate_storage_options,
+            'config.results.storage_options': results_storage_options,
+            'config.temporary.storage_options': temporary_storage_options,
```

Comment:
I don't think there is a `config` key in the config option. Am I missing something?